banyanmonitorgui/banyan_monitor_gui.py

```diff
@@ -198,15 +196,15 @@ def get_message(self):
                     self.incoming_message_processing(data[0].decode(), payload)
                     self.master.after(1, self.get_message)
                 else:
-                    self.incoming_message_processing(data[0].decode(), umsgpack.unpackb(data[1]))
+                    self.incoming_message_processing(data[0].decode(), msgpack.unpackb(data[1]))
+                    self.master.after(1, self.get_message)
+
             except Exception:
                 self.error_reporting()
 
         except zmq.error.Again:
             try:
                 self.master.after(1, self.get_message)
-                time.sleep(self.loop_time)
```

Comment:
Unnecessary line and called sleep within the GUI event loop may cause issues (I am not sure about that though).